### PR TITLE
BROKERS: Update Lab Command

### PR DESCRIPTION
**Pull request by geetika-jain**

### CODE REVIEW

File: DMX.Core.Api/Brokers/Storages/StorageBroker.LabCommands.cs [hunk 1]
```diff
@@ -2,13 +2,27 @@
...
+using Microsoft.EntityFrameworkCore.ChangeTracking;
 
 namespace DMX.Core.Api.Brokers.Storages
 {
     public partial class StorageBroker
     {
         public DbSet<LabCommand> LabCommands { get; set; }
+
+        public async ValueTask<LabCommand> UpdateLabCommand(LabCommand command)
+        {
```
Comment (hassanhabib) [inline DMX.Core.Api/Brokers/Storages/StorageBroker.LabCommands.cs:17]: Should be UpdateLabCommandAsync

Also don't forget the interface method

**Review by hassanhabib — DISMISSED** [review]

File: DMX.Core.Api/Brokers/Storages/IStorageBroker.LabCommands.cs [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+
+using DMX.Core.Api.Models.Foundations.LabCommands;
+using System.Threading.Tasks;
+
+namespace DMX.Core.Api.Brokers.Storages
+{
+    public partial interface IStorageBroker
+    {
+        ValueTask<LabCommand> UpdateLabCommandAsync(LabCommand command);
+        ValueTask<LabCommand> InsertLabCommandAsync(LabCommand labCommand);
```
Comment (hassanhabib) [inline DMX.Core.Api/Brokers/Storages/IStorageBroker.LabCommands.cs:13]: Honor the `CRUD` order - so `Insert` comes before `Update`

File: DMX.Core.Api/Brokers/Storages/StorageBroker.LabCommands.cs [hunk 3]
```diff
@@ -13,7 +13,14 @@ namespace DMX.Core.Api.Brokers.Storages
...
     {
         public DbSet<LabCommand> LabCommands { get; set; }
+        
+        public async ValueTask<LabCommand> SelectLabCommandByIdAsync(Guid labCommandId)
+        {
+            var broker = new StorageBroker(this.configuration);
 
+            return await broker.LabCommands.FindAsync(labCommandId);
+        }
+
```
Comment (hassanhabib) [inline DMX.Core.Api/Brokers/Storages/StorageBroker.LabCommands.cs:23]: Honor the same `CRUD` order here as well @geetika-jain

**Review by geetika-jain — COMMENTED** [review]
CODE RUB: Edit order as per CRUD

**Review by hassanhabib — APPROVED** [review]

**Pull request merged**